### ISSUE TITLE
How to insert some seeds for testing fast

### ISSUE THREAD

**Issue by 4ydan** [ElektraInitiative/PermaplanT#633]
I have the following problem.
So I like to wipe my environment a lot of times. My database is currently empty and I need a fast way to add a couple of seeds for manual testing. I have no idea how to get them from nextcloud and running the scrappers takes infinite amount of time. What am I missing.

**Comment by markus2330** [comment]
Good idea, we can provide smaller plant CSVs. E.g. only tomato, basil and garlic :wink:

**Comment by markus2330** [comment]
See NC chat, it was created. It will need an update after #479, so I keep this open.

**Comment by temmey** [comment]
just using npm run insert (only inserts `mergedDatasets.csv` till #606 is accepted) takes 12.5sec on my machine. 
Is this not fast enough?

**Comment by markus2330** [comment]
I thought so, too. But now we already have it...

**Comment by 4ydan** [comment]
But is that with mergedData downloaded from NC?

**Comment by markus2330** [comment]
Yes, all data sets are in NC.

**Comment by 4ydan** [comment]
But NC is not available to everyone, so when a new contributor comes to this project, he doesn't have any seeds, also it's quite a big entry barrier and requires some additional steps to setup the seeds.

**Comment by markus2330** [comment]
Contributors get access to NC. Unfortunately, we cannot make the data public due to licensing (#102 and more).

**Comment by markus2330** [comment]
No, it is important that devs have realistic data, so please use the data in NC. NC is a very flexible file sync tool, it can automatically sync, maybe look into it a bit: https://nextcloud.com/clients/

**Comment by markus2330** [comment]
Thx, will close. About updating the docu we have #634.
